File: Introduction to Computer Programming - CSYS 1203/Assignment 3/average.py
# This simple program will average the numbers entered by a user.
# By Michael McQuade CSYS1203

def main():
    print("This program will average comma separated numbers you enter.")
    n = eval(input("How many numbers are to be averaged? "))
    avg = eval(input("Please enter the numbers you would like averaged: "))

    avg = sum(avg) / n
    print(avg)


main()

File: Computer Science II - CS 2133/Assignment 1/2 Statistics/Average.java
/**
 * Created by giraffe on 8/29/2016.
 */

import java.util.Scanner;

public class Average {
    public static void main(String[] args)
    {
        int currentInput;
        int runningSum = 0;
        int countOfNums = 0;
        Scanner userInput = new Scanner(System.in);


        System.out.println("Please enter as many numbers as you would like to be averaged, type a negative number to quit: ");


        while(true)
        {
            currentInput = userInput.nextInt();

            if(currentInput < 0)
            {
                break;
            }
            runningSum += currentInput;
            countOfNums++;


        }
        System.out.println("You entered " + countOfNums + " numbers. The average is: " + runningSum / countOfNums);
    }
}
